public/new-image-card.html

```diff
@@ -169,14 +173,13 @@
           this.handleFileInput_('styleFile_', this.$.styleFileInput.files, this.$.styleImagePreview);
         },
         handleFileInput_: function(fileProperty, files, imageElement) {
-          this.set(fileProperty, null);
-          if (files.length != 1) {
-            return;
-          }
-          if (!files[0].type.startsWith('image/')) {
-            return;
+          for (var i = 0; i < files.length; i++) {
+            if (!files[i].type.startsWith('image/')) {
+              return;
+            }
           }
-          this.set(fileProperty, files[0]);
+          this.set(fileProperty, []);
```

Comment:
Do this at the top of the function (above the image type check) to match the previous behavior